docs/pom.md

```diff
@@ -0,0 +1,30 @@
+The following metadata fields can be extracted from a pom.xml file
```

Comment:
Also add here that this was based on the CodeMeta crosswalk from the pom.xml (link)